port/common/omrport.c

```diff
@@ -250,6 +250,10 @@ static OMRPortLibrary MasterPortLibraryTable = {
 	omrsysinfo_get_os_description, /* sysinfo_get_os_description */
 	omrsysinfo_os_has_feature, /* sysinfo_os_has_feature */
 	omrsysinfo_os_kernel_info, /* sysinfo_os_kernel_info */
+	omrsysinfo_is_cgroup_limits_supported, /* sysinfo_is_cgroup_limits_supported */
+	omrsysinfo_is_cgroup_limits_enabled, /* sysinfo_is_cgroup_limits_enabled */
+	omrsysinfo_enable_cgroup_limits, /* sysinfo_enable_cgroup_limits */
+	omrsysinfo_cgroup_get_memlimit, /* sysinfo_cgroup_get_memlimit */	
```

Comment:
To namespace this more consistently, and to make it easier to search the code base for cgroup related functions, can the names be changed to:
```
omrsysinfo_cgroup_is_limits_supported, /* sysinfo_is_cgroup_limits_supported */
omrsysinfo_cgroup_is_limits_enabled, /* sysinfo_is_cgroup_limits_enabled */
omrsysinfo_cgroup_enable_limits, /* sysinfo_enable_cgroup_limits */
omrsysinfo_cgroup_get_memlimit, /* sysinfo_cgroup_get_memlimit */
```